app/mm-redux/client/client4.ts

```diff
@@ -68,7 +68,7 @@ export default class Client4 {
     }
 
     setUrl(url: string) {
-        this.url = url;
+        this.url = url.replace(/\/+$/, '');
```

Comment:
This regex allows subpaths?

Reply:
Yes it just removes the trailing slash and shouldn't affect the subpaths.

`http://localhost:8065/company/mattermost/` becomes `http://localhost:8065/company/mattermost`

Reply:
Maybe just add it to the test ;)